### PR TITLE
Add HTTP Archive entry to In the Wild page

### PR DESCRIPTION
**Pull request by tunetheweb**
Also did some other clean up on the CHANGELOG.

### CODE REVIEW

File: docs/source/inthewild.rst [hunk 1]
```diff
@@ -40,3 +40,10 @@ Just add a section below by raising a PR on Github by
   - Reduced burden on Analytics Engineers to remember every single style rule.
   - New Analytics Engineers can quickly see and learn what "good SQL" looks
     like at Surfline and start writing it from day 1.
+- The `HTTP Archive <https://httparchive.org>` uses SQL Fluff to automatically
```
Comment (barrywhart) [inline docs/source/inthewild.rst:43]:
```suggestion
- The `HTTP Archive <https://httparchive.org>` uses SQLFluff to automatically
```

File: docs/source/inthewild.rst [hunk 2]
```diff
@@ -40,3 +40,10 @@ Just add a section below by raising a PR on Github by
...
+  check for quality and consistency of code submitted by the many contributors
+  to this project. In particular our annual `Web Almanac <https://almanac.httparchive.org>`
+  attracts hundreds of volunteers to help analyse our BigQuery dataset and
+  being able automatically lint Pull Requests through GitHub Actions is a
+  fantastic way to help us maintain repositary of
```
Comment (barrywhart) [inline docs/source/inthewild.rst:48]:
```suggestion
  fantastic way to help us maintain a repository of
```

**Review by barrywhart — CHANGES_REQUESTED** [review]

File: docs/source/inthewild.rst (hunk 1 above)
Comment (tunetheweb) [reply docs/source/inthewild.rst]: Thanks - fixed!

File: docs/source/inthewild.rst (hunk 2 above)
Comment (tunetheweb) [reply docs/source/inthewild.rst]: Fixed in https://github.com/sqlfluff/sqlfluff/pull/1267/commits/2836ce26d061bb79c1f805babc54fb9cecf24eae (note: slightly different than your suggestion).

**Review by barrywhart — APPROVED** [review]

**Pull request merged**